substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/VMInspectionOptions.java

```diff
@@ -60,11 +60,13 @@ public final class VMInspectionOptions {
     private static final String MONITORING_JMXSERVER_NAME = "jmxserver";
     private static final String MONITORING_THREADDUMP_NAME = "threaddump";
     private static final String MONITORING_NMT_NAME = "nmt";
+    private static final String MONITORING_GC_NOTIFICATIONS = "gcnotif";
```

Comment:
I will need to discuss the option name internally. Does HotSpot use the term `gcnotif` in any way (e.g., is there an option to disable GC notifications)?

Reply:
Hotspot doesn't use the name `gcnotif`. I just decided on it arbitrarily. As far as I can tell, GC notifications are always are enabled (as long as there are some listeners registered).